interface/forms_admin/forms_admin.php

```diff
@@ -46,195 +46,217 @@
+                                } else {
+                                    echo "<td><a class='link_submit text-success' href='./forms_admin.php?id=" . attr_url($registry['id']) . "&method=disable&csrf_token_form=" . attr_url(CsrfUtils::collectCsrfToken()) . "'>" . xlt('enabled') . "</a>";
+                                }
+                                ?>
+                                </td>
+                                <td class="<?php echo attr($tdStyle); ?>">
+                                    <span class='text'><?php
+                                        if ($registry['unpackaged']) {
+                                            echo xlt('PHP extracted');
+                                        } else {
+                                            echo xlt('PHP compressed');
+                                        }
+                                        ?>
+                                    </span>
+                                </td>
+                                <td class="<?php echo attr($tdStyle); ?>">
+                                    <?php
+                                    if ($registry['sql_run']) {
+                                        echo "<span class='text'>" . xlt('DB installed') . "</span>";
+                                    } else {
+                                        echo "<a class='link_submit' href='./forms_admin.php?id=" . attr_url($registry['id']) . "&method=install_db&csrf_token_form=" . attr_url(CsrfUtils::collectCsrfToken()) . "'>" . xlt('install DB') . "</a>";
+                                    }
+                                    ?>
+                                </td>
+                                <?php
+                                echo "<td><input type='text' size='4'  name='priority_" . attr($registry['id']) . "' value='" . attr($priority_category['priority']) . "'></td>";
+                                echo "<td><input type='text' size='10' name='category_" . attr($registry['id']) . "' value='" . attr($priority_category['category']) . "'></td>";
+                                echo "<td><input type='text' size='10' name='nickname_" . attr($registry['id']) . "' value='" . attr($priority_category['nickname']) . "'></td>";
+                                echo "<td>";
+                                echo "<select name='aco_spec_" . attr($registry['id']) . "'>";
+                                echo "<option value=''></option>";
+                                echo AclExtended::genAcoHtmlOptions($priority_category['aco_spec']);
+                                echo "</select>";
+                                echo "</td>";
+                                ?>
+                            </tr>
+                                <?php
+                                if ($tdStyle=="bg-dark text-light") {
```

Comment:
$tdStyle = $tdStyle == "bg-dark text-light" ? "bg-light" : "bg-dark text-light";

Just a guy in love with ternary operator :)

Reply:
The ternary operator is a good choice,
But I just follow the same coding style lol